optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/__init__.py

```diff
@@ -152,93 +150,90 @@ def _handle_control_tag_using(path, name, using, value):
     return using
 
 
-def _apply_using(name, cls_node, using, node):
```

Comment:
I cant see how `cls_node` is related to this commit message. Maybe just make it clear on the commit message.

Reply:
Oh, `cls_node` is the thing that allows one to use the debug version of the Tree Node class.